core/templates/components/state-directives/hint-editor/hint-editor.component.html

```diff
@@ -59,3 +63,13 @@
     </div>
   </div>
 </div>
+
+<style>
+  .oppia-rule-preview {
+    position: relative;
+  }
+
+  .oppia-click-to-start-editing {
+    position: absolute;
+  }
```

Comment:
Please make sure to test the mobile UI as well, in case you haven't already.

Reply:
okay

Reply:
![kevin](https://user-images.githubusercontent.com/77006897/148691303-c0c48fef-8858-4f92-b53c-a26169d4b34e.png)

Reply:
Proof*

Reply:
great, thanks!